aws/GitOps/clusters/optional-test-infra-prow/namespaces/prow/config.yaml

```diff
@@ -136,9 +136,7 @@ data:
       kubeflow/kubeflow:
       - name: kubeflow-kubeflow-presubmit  # convention: (job type)-(repo name)-(suite name)-(test type)
         branches:
-        - master
-        - v1.3-branch
-        - v1.4-branch
+        - ^master|v.+$
```

Comment:
Should it be `^master|v.+-branch$`? Otherwise it will trigger prow job on any branch starting with `v`.

Reply:
That's ideal, done!